src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -1,668 +1,458 @@
-rendering on the screen.
-
-<picture>
-  <source srcset="{{ "image/tcFciHGuF3MxnTr1y5ue01OGLBn2/elqsdYqQEefWJbUM2qMO.svg" | imgix }}" media="(min-width: 640px)">
-  {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/9trpfS9wruEPGekHqBdn.svg", alt="Good LCP values are 2.5 seconds, poor values are greater than 4.0 seconds and anything in between needs improvement", width="384", height="96" %}
-</picture>
-
-The most common causes of a poor LCP are:
-
-- [Slow server response times](#slow-servers)
-- [Render-blocking JavaScript and CSS](#render-blocking-resources)
-- [Slow resource load times](#slow-resource-load-times)
-- [Client-side rendering](#client-side-rendering)
-
-## Slow server response times {: #slow-servers }
-
-The longer it takes a browser to receive content from the server, the longer it
-takes to render anything on the screen. A faster server response time directly
-improves every single page-load metric, including LCP.
-
-Before anything else, improve how and where your server handles your content.
-Use [**Time to First Byte**](/ttfb/) (TTFB) to measure your server
-response times. You can improve your TTFB in a number of different ways:
-
-- Optimize your server
-- Route users to a nearby CDN
-- Cache assets
-- Serve HTML pages cache-first
-- Establish third-party connections early
-- Use signed exchanges
-
-### Optimize your server
-
-Are you running expensive queries that take your server a significant amount of
-time to complete? Or are there other complex operations happening server-side
-that delay the process to return page content? Analyzing and improving the
-efficiency of your server-side code will directly improve the time it takes for
-the browser to receive the data.
-
-Instead of just immediately serving a static page on a browser request, many
-server-side web frameworks need to create the web page dynamically. In other
-words, rather than just sending a complete HTML file that's already ready when
-the browser requests it, the frameworks need to run logic to construct the page.
-This could be due to pending results from a database query or even because
-components need to be generated into markup by a UI framework (such as
-[React](https://reactjs.org/docs/react-dom-server.html)). Many web frameworks
-that run on the server have performance guidance that you can use to speed up
-this process.
-
-{% Aside %}
-Check out [Fix an overloaded server](/overloaded-server/) for more tips.
-{% endAside %}
+[Largest Contentful Paint (LCP)](/lcp/) is one of the three [Core Web Vitals](/vitals/#core-web-vitals) metrics, and it represents how quickly the main content of a web page is loaded. Specifically, LCP measures the time from when the user initiates loading the page until the largest image or text block is rendered within the viewport.
 
-### Route users to a nearby CDN
+To provide a good user experience, sites should strive to have an LCP of 2.5 seconds or less for at least 75% of page visits.
```

Comment:
```suggestion
To provide a good user experience, **sites should strive to have an LCP of 2.5 seconds or less for at least 75% of page visits.**
```

Reply:
It's a nit, but I think the emphasis helps this jump out of the page so users are more keenly aware of the LCP "good" threshold.